certified-connectors/DocuMotor/apiDefinition.swagger.json

```diff
@@ -0,0 +1,113 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DocuMotor",
+    "description": "Generate documents, presentations, and spreadsheets based on data.\nDocuMotor is a modern document generation platform that allows you to use transformation steps, templates designed in Microsoft 365, and APIs, to create documents based on any data.",
```

Comment:
@sebastianehlers is it possible to remove the word "API" from the description. Please see [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).

Reply:
Adjusted.